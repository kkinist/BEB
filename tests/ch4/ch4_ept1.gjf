%chk=ch4.chk
%nprocs=4
%mem=1000mw
# EPT(OVGF+P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc pop(all,thresh=1)

BEB step 4: EPT B for ch4

0 1

C H 0
6-311+G(d,p)
****

1 4 4 4

