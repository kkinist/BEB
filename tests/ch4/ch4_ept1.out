 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-28593.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     28595.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch4.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ----------------------------------------------------------------------
 # EPT(OVGF+P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc
  pop(all,thresh=1)
 ----------------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,8=3,13=1,38=6/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/11=30100/8;
 6/7=2,8=2,9=2,10=2,87=-1,88=1/1;
 99/5=1,9=1/99;
 -------------------------
 BEB step 4: EPT B for ch4
 -------------------------
 Structure from the checkpoint file:  "ch4.chk"
 Charge =  0 Multiplicity = 1
 Redundant internal coordinates found in file.
 C,0,0.,0.000000001,0.000000001
 H,0,-0.000000006,-0.0000000007,1.0935113774
 H,0,0.,-1.0309724112,-0.3645037927
 H,0,-0.892848297,0.5154862096,-0.3645037952
 H,0,0.892848303,0.5154862062,-0.3645037855
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    0.000000    1.093511
      3          1           0        0.000000   -1.030972   -0.364504
      4          1           0       -0.892848    0.515486   -0.364504
      5          1           0        0.892848    0.515486   -0.364504
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.093511   0.000000
     3  H    1.093511   1.785697   0.000000
     4  H    1.093511   1.785697   1.785697   0.000000
     5  H    1.093511   1.785697   1.785697   1.785697   0.000000
 Stoichiometry    CH4
 Framework group  TD[O(C),4C3(H)]
 Deg. of freedom     1
 Full point group                 TD      NOp  24
 Largest Abelian subgroup         D2      NOp   4
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.631339    0.631339    0.631339
      3          1           0       -0.631339   -0.631339    0.631339
      4          1           0       -0.631339    0.631339   -0.631339
      5          1           0        0.631339   -0.631339   -0.631339
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    157.2594344    157.2594344    157.2594344
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   6 1.00       0.000000000000
      0.4563240000D+04  0.1966650249D-02
      0.6820240000D+03  0.1523060193D-01
      0.1549730000D+03  0.7612690966D-01
      0.4445530000D+02  0.2608010331D+00
      0.1302900000D+02  0.6164620782D+00
      0.1827730000D+01  0.2210060280D+00
 SP   3 1.00       0.000000000000
      0.2096420000D+02  0.1146600807D+00  0.4024869267D-01
      0.4803310000D+01  0.9199996477D+00  0.2375939567D+00
      0.1459330000D+01 -0.3030682134D-02  0.8158538515D+00
 SP   1 1.00       0.000000000000
      0.4834560000D+00  0.1000000000D+01  0.1000000000D+01
 SP   1 1.00       0.000000000000
      0.1455850000D+00  0.1000000000D+01  0.1000000000D+01
 SP   1 1.00       0.000000000000
      0.4380000000D-01  0.1000000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6260000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****
      5 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****

 There are    14 symmetry adapted cartesian basis functions of A   symmetry.
 There are    11 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    11 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    11 symmetry adapted cartesian basis functions of B3  symmetry.
 There are    13 symmetry adapted basis functions of A   symmetry.
 There are    11 symmetry adapted basis functions of B1  symmetry.
 There are    11 symmetry adapted basis functions of B2  symmetry.
 There are    11 symmetry adapted basis functions of B3  symmetry.
    46 basis functions,    68 primitive gaussians,    47 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy        13.3922467928 Hartrees.
 NAtoms=    5 NActive=    5 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    46 RedAO= T EigKep=  9.79D-03  NBF=    13    11    11    11
 NBsUse=    46 1.00D-06 EigRej= -1.00D+00 NBFU=    13    11    11    11
 Initial guess from the checkpoint file:  "ch4.chk"
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
       Occupied  (A1) (A1) (T2) (T2) (T2)
       Virtual   (A1) (T2) (T2) (T2) (T2) (T2) (T2) (A1) (T2) (T2)
                 (T2) (A1) (E) (E) (T2) (T2) (T2) (T1) (T1) (T1)
                 (T2) (T2) (T2) (A1) (E) (E) (T2) (T2) (T2) (T2)
                 (T2) (T2) (A1) (T2) (T2) (T2) (A1) (A1) (T2) (T2)
                 (T2)
 Keep R1 ints in memory in symmetry-blocked form, NReq=1439884.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RHF) =  -40.2088631777     A.U. after    7 cycles
            NFock=  7  Conv=0.36D-08     -V/T= 2.0019
 QCSCF skips out because SCF is already converged.
 ExpMin= 4.38D-02 ExpMax= 4.56D+03 ExpMxC= 6.82D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    46
 NBasis=    46 NAE=     5 NBE=     5 NFC=     1 NFV=     0
 NROrb=     45 NOA=     4 NOB=     4 NVA=    41 NVB=    41
 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048377173
 LASXX=        41859 LTotXX=       41859 LenRXX=       86506
 LTotAB=       44647 MaxLAS=       98820 LenRXY=           0
 NonZer=      128365 LenScr=      785920 LnRSAI=       98820
 LnScr1=      785920 LExtra=           0 Total=      1757166
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=0 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.5462296940D-02 E2=     -0.1602397043D-01
     alpha-beta  T2 =       0.4464051959D-01 E2=     -0.1385878813D+00
     beta-beta   T2 =       0.5462296940D-02 E2=     -0.1602397043D-01
 ANorm=    0.1027406985D+01
 E2 =    -0.1706358222D+00 EUMP2 =    -0.40379498999928D+02
 GREENY: Electron Propagator Program
 FoFJK:  IHMeth= 0 ICntrl=     200 DoSepK=F KAlg= 0 I1Cent=   0 FoldK=F
 IRaf= 990000000 NMat=   1 IRICut=       1 DoRegI=T DoRafI=F ISym2E=-1.
 FoFCou: FMM=F IPFlag=           0 FMFlag=           0 FMFlg1=           0
         NFxFlg=           0 DoJE=F BraDBF=F KetDBF=F FulRan=T
         wScrn=  0.000000 ICntrl=     200 IOpCl=  0 I1Cent=           0 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.

 Summary of results for alpha spin-orbital   1  OVGF: 
 Koopmans theorem:            -0.94133D+00 au  -25.615 eV
 Converged second order pole: -0.85565D+00 au  -23.284 eV  0.883 (PS)
 Converged third  order pole: -0.86300D+00 au  -23.483 eV  0.876 (PS)
 Outer Valence Approximation: -0.86073D+00 au  -23.422 eV  0.872 (PS)

 Summary of results for alpha spin-orbital   1    P3: 
 Koopmans theorem:            -0.94133D+00 au  -25.615 eV
 Converged second order pole: -0.85565D+00 au  -23.284 eV  0.883 (PS)
 Converged 3rd order P3 pole: -0.85234D+00 au  -23.193 eV  0.868 (PS)

 Summary of results for alpha spin-orbital   2  OVGF: 
 Koopmans theorem:            -0.54379D+00 au  -14.797 eV
 Converged second order pole: -0.51127D+00 au  -13.912 eV  0.922 (PS)
 Converged third  order pole: -0.52464D+00 au  -14.276 eV  0.932 (PS)
 Outer Valence Approximation: -0.52283D+00 au  -14.227 eV  0.928 (PS)

 Summary of results for alpha spin-orbital   2    P3: 
 Koopmans theorem:            -0.54379D+00 au  -14.797 eV
 Converged second order pole: -0.51127D+00 au  -13.912 eV  0.922 (PS)
 Converged 3rd order P3 pole: -0.51703D+00 au  -14.069 eV  0.927 (PS)
 Orbitals  3 and  2 are degenerate.  Skipping orbital  3

 Orbitals  4 and  2 are degenerate.  Skipping orbital  4


 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A1) (A1) (T2) (T2) (T2)
       Virtual   (A1) (T2) (T2) (T2) (T2) (T2) (T2) (A1) (T2) (T2)
                 (T2) (A1) (T2) (T2) (T2) (A1) (E) (E) (T1) (T1)
                 (T1) (T2) (T2) (T2) (T2) (T2) (T2) (E) (E) (A1)
                 (T2) (T2) (T2) (T2) (T2) (T2) (A1) (T2) (T2) (T2)
                 (A1)
 The electronic state is 1-A1.
 Alpha  occ. eigenvalues --  -11.20987  -0.94133  -0.54379  -0.54379  -0.54379
 Alpha virt. eigenvalues --    0.07406   0.11025   0.11025   0.11025   0.27095
 Alpha virt. eigenvalues --    0.27095   0.27095   0.31433   0.53495   0.53495
 Alpha virt. eigenvalues --    0.53495   0.79472   0.81550   0.81550   0.81550
 Alpha virt. eigenvalues --    1.03123   1.20165   1.20165   1.72055   1.72055
 Alpha virt. eigenvalues --    1.72055   1.79204   1.79204   1.79204   2.19085
 Alpha virt. eigenvalues --    2.19085   2.19085   2.61522   2.61522   2.61549
 Alpha virt. eigenvalues --    2.78856   2.78856   2.78856   2.90987   2.90987
 Alpha virt. eigenvalues --    2.90987   3.19776   4.18682   4.18682   4.18682
 Alpha virt. eigenvalues --   25.17731
          Condensed to atoms (all electrons):
               1          2          3          4          5
     1  C    4.802181   0.407445   0.407445   0.407445   0.407445
     2  H    0.407445   0.560994  -0.025476  -0.025476  -0.025476
     3  H    0.407445  -0.025476   0.560994  -0.025476  -0.025476
     4  H    0.407445  -0.025476  -0.025476   0.560994  -0.025476
     5  H    0.407445  -0.025476  -0.025476  -0.025476   0.560994
 Mulliken charges:
               1
     1  C   -0.431962
     2  H    0.107990
     3  H    0.107990
     4  H    0.107990
     5  H    0.107990
 Sum of Mulliken charges =   0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.000000
 Electronic spatial extent (au):  <R**2>=             35.9142
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -8.4439   YY=             -8.4439   ZZ=             -8.4439
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              0.0000   YY=              0.0000   ZZ=              0.0000
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.7206
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -16.6951 YYYY=            -16.6951 ZZZZ=            -16.6951 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -5.0387 XXZZ=             -5.0387 YYZZ=             -5.0387
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 1.339224679282D+01 E-N=-1.197693071688D+02  KE= 4.013219651347D+01
 Symmetry A    KE= 3.446067148180D+01
 Symmetry B1   KE= 1.890508343890D+00
 Symmetry B2   KE= 1.890508343890D+00
 Symmetry B3   KE= 1.890508343890D+00

 Atomic contributions to Alpha molecular orbitals:
 Alpha occ 1 OE=-11.210 is C1-s=1.00
 Alpha occ 2 OE=-0.941 is C1-s=0.63 H2-s=0.09 H3-s=0.09 H5-s=0.09 H4-s=0.09
 Alpha occ 3 OE=-0.544 is C1-p=0.52 H2-s=0.11 H3-s=0.11 H5-s=0.11 H4-s=0.11
 Alpha occ 4 OE=-0.544 is C1-p=0.52 H2-s=0.11 H5-s=0.11 H3-s=0.11 H4-s=0.11
 Alpha occ 5 OE=-0.544 is C1-p=0.52 H5-s=0.11 H4-s=0.11 H3-s=0.11 H2-s=0.11
 Alpha vir 6 OE=0.074 is C1-s=1.94 H2-s=-0.24 H4-s=-0.24 H5-s=-0.24 H3-s=-0.24
 Alpha vir 7 OE=0.110 is C1-p=1.28 H4-s=-0.07 H3-s=-0.07 H2-s=-0.07 H5-s=-0.07
 Alpha vir 8 OE=0.110 is C1-p=1.28 H3-s=-0.07 H4-s=-0.07 H2-s=-0.07 H5-s=-0.07
 Alpha vir 9 OE=0.110 is C1-p=1.28 H5-s=-0.07 H4-s=-0.07 H2-s=-0.07 H3-s=-0.07
 Alpha vir 10 OE=0.271 is H5-s=0.26 H4-s=0.26 H3-s=0.26 H2-s=0.26 C1-p=-0.07
 Alpha vir 11 OE=0.271 is H4-s=0.26 H5-s=0.26 H2-s=0.26 H3-s=0.26 C1-p=-0.07
 Alpha vir 12 OE=0.271 is H4-s=0.26 H5-s=0.26 H2-s=0.26 H3-s=0.26 C1-p=-0.07
 Alpha vir 13 OE=0.314 is C1-s=-1.41 H3-s=0.60 H2-s=0.60 H5-s=0.60 H4-s=0.60
 Alpha vir 14 OE=0.535 is C1-p=0.38 H4-s=0.16 H2-s=0.16 H3-s=0.16 H5-s=0.16 C1-d=-0.02
 Alpha vir 15 OE=0.535 is C1-p=0.38 H4-s=0.16 H5-s=0.16 H3-s=0.16 H2-s=0.16 C1-d=-0.02
 Alpha vir 16 OE=0.535 is C1-p=0.38 H5-s=0.16 H4-s=0.16 H2-s=0.16 H3-s=0.16 C1-d=-0.02
 Alpha vir 17 OE=0.795 is C1-s=0.29 H5-s=0.18 H4-s=0.18 H2-s=0.18 H3-s=0.18
 Alpha vir 18 OE=0.816 is C1-p=0.86 H5-s=0.02 H3-s=0.02 H4-s=0.02 H2-s=0.02 C1-d=0.02 H2-p=0.01 H3-p=0.01 H5-p=0.01 H4-p=0.01
 Alpha vir 19 OE=0.816 is C1-p=0.86 H3-s=0.02 H2-s=0.02 H4-s=0.02 H5-s=0.02 C1-d=0.02 H2-p=0.01 H5-p=0.01 H4-p=0.01 H3-p=0.01
 Alpha vir 20 OE=0.816 is C1-p=0.86 H5-s=0.02 H3-s=0.02 H2-s=0.02 H4-s=0.02 C1-d=0.02 H2-p=0.01 H3-p=0.01 H5-p=0.01 H4-p=0.01
 Alpha vir 21 OE=1.031 is C1-s=1.50 H2-s=-0.15 H5-s=-0.15 H3-s=-0.15 H4-s=-0.15 H3-p=0.02 H2-p=0.02 H4-p=0.02 H5-p=0.02
 Alpha vir 22 OE=1.202 is C1-d=0.41 H4-p=0.15 H5-p=0.15 H2-p=0.15 H3-p=0.15
 Alpha vir 23 OE=1.202 is C1-d=0.41 H3-p=0.15 H2-p=0.15 H5-p=0.15 H4-p=0.15
 Alpha vir 24 OE=1.721 is H4-p=0.25 H2-p=0.25 H5-p=0.25 H3-p=0.25
 Alpha vir 25 OE=1.721 is H2-p=0.25 H3-p=0.25 H5-p=0.25 H4-p=0.25
 Alpha vir 26 OE=1.721 is H2-p=0.25 H4-p=0.25 H3-p=0.25 H5-p=0.25
 Alpha vir 27 OE=1.792 is C1-d=0.53 H2-p=0.09 H3-p=0.09 H4-p=0.09 H5-p=0.09 C1-p=-0.05 H4-s=0.04 H5-s=0.04 H3-s=0.04 H2-s=0.04
 Alpha vir 28 OE=1.792 is C1-d=0.53 H3-p=0.09 H4-p=0.09 H5-p=0.09 H2-p=0.09 C1-p=-0.05 H4-s=0.04 H5-s=0.04 H2-s=0.04 H3-s=0.04
 Alpha vir 29 OE=1.792 is C1-d=0.53 H4-p=0.09 H5-p=0.09 H2-p=0.09 H3-p=0.09 C1-p=-0.05 H5-s=0.04 H4-s=0.04 H3-s=0.04 H2-s=0.04
 Alpha vir 30 OE=2.191 is C1-d=0.20 H5-p=0.15 H2-p=0.15 H3-p=0.15 H4-p=0.15 C1-p=0.07 H5-s=0.03 H3-s=0.03 H2-s=0.03 H4-s=0.03
 Alpha vir 31 OE=2.191 is C1-d=0.20 H5-p=0.15 H4-p=0.15 H2-p=0.15 H3-p=0.15 C1-p=0.07 H3-s=0.03 H2-s=0.03 H5-s=0.03 H4-s=0.03
 Alpha vir 32 OE=2.191 is C1-d=0.20 H5-p=0.15 H4-p=0.15 H2-p=0.15 H3-p=0.15 C1-p=0.07 H2-s=0.03 H3-s=0.03 H4-s=0.03 H5-s=0.03
 Alpha vir 33 OE=2.615 is C1-d=0.59 H2-p=0.10 H3-p=0.10 H4-p=0.10 H5-p=0.10
 Alpha vir 34 OE=2.615 is C1-d=0.59 H4-p=0.10 H5-p=0.10 H2-p=0.10 H3-p=0.10
 Alpha vir 35 OE=2.615 is H2-s=0.17 H3-s=0.17 H4-s=0.17 H5-s=0.17 C1-s=0.14 H3-p=0.05 H2-p=0.05 H4-p=0.05 H5-p=0.05
 Alpha vir 36 OE=2.789 is H2-p=0.22 H3-p=0.22 H4-p=0.22 H5-p=0.22 C1-p=-0.14 H5-s=0.09 H2-s=0.09 H3-s=0.09 H4-s=0.09 C1-d=-0.08
 Alpha vir 37 OE=2.789 is H3-p=0.22 H2-p=0.22 H4-p=0.22 H5-p=0.22 C1-p=-0.14 H5-s=0.09 H2-s=0.09 H3-s=0.09 H4-s=0.09 C1-d=-0.08
 Alpha vir 38 OE=2.789 is H2-p=0.22 H3-p=0.22 H5-p=0.22 H4-p=0.22 C1-p=-0.14 H4-s=0.09 H5-s=0.09 H2-s=0.09 H3-s=0.09 C1-d=-0.08
 Alpha vir 39 OE=2.910 is C1-p=0.37 C1-d=0.17 H4-s=0.09 H3-s=0.09 H5-s=0.09 H2-s=0.09 H3-p=0.02 H2-p=0.02 H5-p=0.02 H4-p=0.02
 Alpha vir 40 OE=2.910 is C1-p=0.37 C1-d=0.17 H5-s=0.09 H3-s=0.09 H2-s=0.09 H4-s=0.09 H4-p=0.02 H5-p=0.02 H2-p=0.02 H3-p=0.02
 Alpha vir 41 OE=2.910 is C1-p=0.37 C1-d=0.17 H2-s=0.09 H3-s=0.09 H5-s=0.09 H4-s=0.09 H2-p=0.02 H5-p=0.02 H4-p=0.02 H3-p=0.02
 Alpha vir 42 OE=3.198 is H3-p=0.17 H2-p=0.17 H4-p=0.17 H5-p=0.17 H3-s=0.09 H4-s=0.09 H5-s=0.09 H2-s=0.09 C1-s=-0.08
 Alpha vir 43 OE=4.187 is C1-p=0.79 C1-d=0.16
 Alpha vir 44 OE=4.187 is C1-p=0.79 C1-d=0.16
 Alpha vir 45 OE=4.187 is C1-p=0.79 C1-d=0.16
 Alpha vir 46 OE=25.177 is C1-s=0.99
 
 1\1\GINC-GAMBA\SP\ROVGF-FC\Gen\C1H4\IRIKURA\17-May-2017\0\\# EPT(OVGF+
 P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc pop(all,th
 resh=1)\\BEB step 4: EPT B for ch4\\0,1\C,0,0.,0.000000001,0.000000001
 \H,0,-0.000000006,-0.0000000007,1.0935113774\H,0,0.,-1.0309724112,-0.3
 645037927\H,0,-0.892848297,0.5154862096,-0.3645037952\H,0,0.892848303,
 0.5154862062,-0.3645037855\\Version=ES64L-G09RevD.01\State=1-A1\HF=-40
 .2088632\MP2=-40.379499\RMSD=3.631e-09\PG=TD [O(C1),4C3(H1)]\\@


 SIGN SEEN IN A RESTAURANT-
 WE RESERVE THE RIGHT TO SERVE REFUSE TO ANYONE.
 Job cpu time:       0 days  0 hours  0 minutes 19.6 seconds.
 File lengths (MBytes):  RWF=     20 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:20:43 2017.
